.github/workflows/wheels.yml

```diff
@@ -59,7 +59,7 @@ jobs:
 
     # Enable tmate debugging of manually-triggered workflows if the input option was provided
     - name: Setup tmate session
-      if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.debug_enabled }}
+      if: ${{ always() && github.event_name == 'workflow_dispatch' && github.event.inputs.debug_enabled }}
```

Comment:
does this do anything? seems like `if true && $other_statements` is equivalent to `if $other_statements`.

Reply:
I thought that too, but according to this it does: https://github.community/t/multiple-if-conditional-with-always-is-always-true/128840/2. But I haven't been able to actually try it yet as the change needs to be merged into the main branch first...